Update/some_004.txt

```diff
@@ -3444,30 +3444,30 @@ void main()
 //r…でも、どうしてだろう。それをぎゅっ、と跡が残るくらいに握り締めながら、…千紗登ちゃんは心の奥底まで射抜くような視線で、私をじっと見つめていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…でも、どうしてだろう。それをぎゅっ、と跡が残るくらいに握り締めながら、…千紗登ちゃんは心の奥底まで射抜くような視線で、私をじっと見つめていた。",
-			NULL, "...But, why? Chisato-chan was squeezing the bag as tightly as possible... and staring at me with a gaze that pierced into my heart.", Line_Normal);
+			NULL, "...But, why? Chisato-chan was clutching bag so tightly it was crumpling... staring at me motionlessly with a gaze that pierced through my heart.", Line_Normal);
 	ClearMessage();
 
 //千紗登rvS04/37/310300296.「…遅くなったね。kvS04/37/310300297.結構待った？」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300296", 256, TRUE);
 	OutputLine(NULL, "「…遅くなったね。",
-			NULL, "\"...I'm late.", Line_WaitForInput);
+			NULL, "\"...I'm sorry I'm late.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300297", 256, TRUE);
 	OutputLine(NULL, "結構待った？」",
-			NULL, " Were you waiting a while?\"", Line_Normal);
+			NULL, " Did you wait a while?\"", Line_Normal);
 	ClearMessage();
 
 //夏美rvS04/36/310100417.「う、ううん…」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 36, "ps3/s04/36/310100417", 256, TRUE);
 	OutputLine(NULL, "「う、ううん…」",
-			NULL, "\"N-no...\"", Line_Normal);
+			NULL, "\"N-No...\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"N-no...\"", Line_Normal);
```
Another one of those far-reaching changes. Style for stammering throughout the console arcs (and, as far as I know, the main arcs) is initial cap followed by lowercase. If we change here, we'll need to change everywhere. Up to the team!